spec/spec_helper.rb

```diff
@@ -10,7 +10,7 @@
 require 'pry-byebug'
 
 Dir['./spec/support/**/*.rb'].each { |f| require f }
-
+# RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = 999_999_999
```

Comment:
You seem to have used this for debugging - interesting. I either never came across a use case for increasing the value or didn't know about that. Just leave it here if it is useful 👍

Reply:
It's helpful if rspec compares complex objects, because it will start to truncate at some point to keep the output consistent

I wanted to keep it in commented, because I always have to google the exact variable name otherwise